promptulate/config.py

```diff
@@ -129,6 +129,33 @@ def get_ernie_api_secret(self) -> str:
             return get_cache()["ERNIE_API_SECRET"]
         raise ValueError("ERNIE_API_SECRET is not provided. Please set your secret.")
 
+    def get_zhipuai_api_(self) -> str:
```

Comment:
In the future version, I think we can use pne_cache.get(key) to get pne cache.

Reply:
ok